reframe/frontend/executors/policies.py

```diff
@@ -227,6 +233,9 @@ def __init__(self):
         # Index tasks by test cases
         self._task_index = {}
 
+        # All currently building tasks per partition
+        self._building_tasks = {}
+
         # All currently running tasks per partition
```

Comment:
Is this now accurate? I think that the running tasks are all those in the run phase.